src/com/mambu/apisdk/services/ClientsService.java

```diff
@@ -176,49 +176,32 @@ public Client getClient(String clientId) throws MambuApiException {
 	}
 
 	/**
-	 * @deprecated As of release 3.8, replaced by {@link #getClients(boolean, int, int)}
-	 * 
-	 *             Requests a list of all matching clients.
-	 * 
-	 * @param active
-	 *            True if active Clients should retrieved, false for inactive Clients
-	 * 
-	 * @return the list of Mambu clients
-	 * 
-	 * @throws MambuApiException
-	 */
-	@Deprecated
-	public List<Client> getClients(boolean active) throws MambuApiException {
-
-		ParamsMap params = new ParamsMap();
-		params.addParam(CLIENT_STATE, (active ? "ACTIVE" : "INACTIVE"));
-
-		return serviceExecutor.execute(getClientsList, params);
-	}
-
-	/**
 	 * Requests a list of clients, limited by offset/limit
 	 * 
 	 * @param active
 	 *            True if active Clients should retrieved, false for inactive Clients
 	 * @param offset
-	 *            Offset to start loading Clients, has to be >= 0
+	 *            Offset to start loading Clients, has to be >= 0 if not null

```

Comment:
Please note here and below that the default offset and limit will be used if it's set to null. Otherwise one might still think all clients are returned.